### DISCUSSION TITLE
Make Room-Assistant reconnect to MQTT

### DISCUSSION THREAD

**Discussion by TCPMeta** [mKeRix/room-assistant#355] [Q&A] [answered]
I ran into an issue doing some Binary Switches so in Home Assistant I removed the Raspberry Pi Zero device that was connected via MQTT and now I can't get it to reconnect. Is there a command on the Pi I should run to make it reconnect/rediscover?

**Comment by mKeRix** [comment, marked as answer]
If it's just about the auto discovery of the devices and entities in Home Assistant: a restart of Home Assistant should do the trick.

Otherwise, room-assistant should auto-reconnect to MQTT. If it doesn't double check the configured credentials and try to restart room-assistant as well.

**Reply by TCPMeta** [reply]
I was able to fix it by rolling back to a previous snapshot backup.
